### PR TITLE
Adding SDS Rollover guide section

### PR DESCRIPTION
**Pull request by AleksMalone-MS**

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9

File: microsoft-365/enterprise/setup-guides-for-microsoft-365.md [hunk 1]
```diff
@@ -245,3 +245,7 @@ For organizations using Configuration Manager, you can use the [Deploy and updat
...
 
 Use the [Intune Configuration Manager co-management setup guide](https://aka.ms/comanagementsetup) to set up existing Configuration Manager client devices and new internet-based devices that your org wants to co-manage with both Microsoft Intune and Configuration Manager. Co-management allows you to manage Windows 10 devices and adds new functionality to your org's devices, while receiving the benefits of both solutions.
+
+#### School Data Sync rollover setup guide
+
+The [SDS Rollover setup guide](https://aka.ms/sdsrolloversetupguide) provides the steps to help your organization sync student information data to Azure Active Directory and Office365. This guide streamlines the term lifecycle management process by creating Office 365 Groups for Exchange Online and Share PointOnline, class teams for Microsoft Teams and One Note, as well as Intune for Education, and rostering and single sign-on integration for third-party apps. You’ll perform end-of-year closeout, tenant cleanup and archive, new school year preparation, and new school year launch. Then you can create new profiles using the sync deployment method that suits your org.
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/setup-guides-for-microsoft-365.md]:
```suggestion
The [SDS Rollover setup guide](https://aka.ms/sdsrolloversetupguide) provides the steps to help your organization sync student information data to Azure Active Directory and Office 365. This guide streamlines the term lifecycle management process by creating Office 365 Groups for Exchange Online and SharePoint Online, class teams for Microsoft Teams and OneNote, as well as Intune for Education, and rostering and single sign-on integration for third-party apps. You’ll perform end-of-year closeout, tenant clean up and archive, new school year preparation, and new school year launch. Then you can create new profiles using the sync deployment method that suits your organization.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @AleksMalone-MS, kindly commit the suggested changes. Many thanks.

**Comment by AleksMalone-MS** [comment]
Hi Johan,

The requested changes have been committed.

Thank you,

Aleks Malone
Microsoft 365 FastTrack IPE - Wizard/Automation - Program Manager
Email: ***@***.******@***.***>
[Logo  Description automatically generated]

Friendly note, my working hours (Monday - Friday 9:00 am  - 5:00 pm (EST), may be different than your working hours. I may not respond right away, and you shouldn't feel the need to respond right away outside of your working hours.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**